MobiFlight/Joysticks/JoystickManager.cs

```diff
@@ -97,20 +98,26 @@ public void Startup()
         public void Shutdown()
         {
             PollTimer.Stop();
-            joysticks.Clear();
+            Joysticks.Clear();
```

Comment:
I think Shutdown should call Stop for all Joysticks to make sure, that they can do their specific stuff. Like turning off the lights or disconnecting from their stream (Octavi)

Reply:
But tbh, we have to provide a Shutdown or Disconnect method for HID devices. In the case of Octavi it opens it's own stream. That's why it keeps sending events even after Shutdown of the JoystickManager